### PR TITLE
Added README.md with project details

### PR DESCRIPTION
**Pull request by Bhavana-Jami**
Hello,
As mentioned in the below issue, 
https://github.com/himanshurelhan70/Codehelp/issues/4#issue-2534110633
I have added details for one project from CSS folder, details like what the project is about, project structure, technologies used and how to run it, please review the PR.

### CODE REVIEW

File: Projects/CSS/01_Card/README.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+│   ├── thumbnail-1.jpg # Thumbnail image for the Web Development course</br>
+│   ├── thumbnail-2.jpg # Thumbnail image for the DSA course</br>
+│   └── thumbnail-3.jpg # Thumbnail image for the OOPs course</br>
+└── README.md           # Documentation file (you're reading it!)</br>
+# Features
+* Card-Based Layout: The webpage displays each course in a visually appealing card, including an image, title, price, and discount information.
+* Responsive Design: The webpage is built with responsive design principles in mind, adjusting the layout for different screen sizes.
+* Dynamic Links: Each course card links to an external page for the respective course using the target="_blank" attribute, which opens the link in a new tab.
+* Google Fonts Integration: Custom fonts are used via Google Fonts to enhance the typography of the webpage.
+Technology Stack
```
Comment (himanshurelhan70) [inline Projects/CSS/01_Card/README.md:20]: Make `Technology Stack` a heading

File: Projects/CSS/01_Card/README.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+* HTML5: For structuring the content of the webpage.
+* CSS3: Custom CSS is used to design and style the card layout and other elements.
+* Google Fonts: The Rubik font is imported from Google Fonts for a clean, modern look.
+* Static Assets: Images are used for course thumbnails, stored in the images/ directory.
+# How to Run the Project
+* Download or clone this repository: 
+* git clone https://github.com/your-username/codehelp-course-listing.git
+* Open the index.html file in your browser:
```
Comment (himanshurelhan70) [inline Projects/CSS/01_Card/README.md:28]: Make bullet points, something like this

1. Download or clone this repository:
    `git clone https://github.com/your-username/codehelp-course-listing.git`

2. Open the index.html file in your browser:
    - Navigate to the project directory.
    - Open index.html in any web browser to view the course listing page.

File: Projects/CSS/01_Card/README.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+* Dynamic Links: Each course card links to an external page for the respective course using the target="_blank" attribute, which opens the link in a new tab.
+* Google Fonts Integration: Custom fonts are used via Google Fonts to enhance the typography of the webpage.
+Technology Stack
+* HTML5: For structuring the content of the webpage.
+* CSS3: Custom CSS is used to design and style the card layout and other elements.
+* Google Fonts: The Rubik font is imported from Google Fonts for a clean, modern look.
+* Static Assets: Images are used for course thumbnails, stored in the images/ directory.
+# How to Run the Project
+* Download or clone this repository: 
+* git clone https://github.com/your-username/codehelp-course-listing.git
```
Comment (himanshurelhan70) [inline Projects/CSS/01_Card/README.md:27]: Replace `git clone https://github.com/your-username/codehelp-course-listing.git` with `https://github.com/himanshurelhan70/Codehelp.git`

Comment (Bhavana-Jami) [reply Projects/CSS/01_Card/README.md:27]: image
Done

File: Projects/CSS/01_Card/README.md (hunk 2 above)
Comment (Bhavana-Jami) [reply Projects/CSS/01_Card/README.md:28]: image
Done

**Comment by himanshurelhan70** [comment]
Good Work 👍

**Pull request merged**